_maps/yogstation/map_files/YogStation/YogStation.dmm

```diff
@@ -35139,6 +35142,20 @@
 	icon_state = "pipe11-1";
 	dir = 6
 	},
+/obj/structure/closet/secure_closet{
+	name = "psychiatrist locker";
+	req_access_txt = "5"
+	},
+/obj/item/storage/pill_bottle/happiness,
+/obj/item/storage/pill_bottle/floorpill,
```

Comment:
Why?

Reply:
cause imagining the psy giving people who come in a completely random pill is funny

Reply:
Floor pills were meant to be maint only and I am not okay with this